doc/source/upgrading_guide.rst

```diff
@@ -1,22 +1,20 @@
 .. _upgrading_guide:
 
 ==========================================================
- Percona Server In-Place Upgrading Guide: From 5.6 to 5.7
+ Percona Server In-Place Upgrading Guide: From 5.7 to 8.0
 ==========================================================
 
 In-place upgrades are those which are done using the existing data in the server. Generally speaking, this is stopping the server, installing the new server and starting it with the same data files. While they may not be suitable for high-complexity environments, they may be adequate for many scenarios.
 
 The following is a summary of the more relevant changes in the 5.7 series. It's strongly recommended to that you read the following guides as they contain the list of incompatible changes that could cause automatic upgrade to fail: 
 
-  * :ref:`changed_in_57`
-
-  * `Upgrading MySQL <http://dev.mysql.com/doc/refman/5.7/en/upgrading.html>`_
-
-  * `Upgrading from MySQL 5.6 to 5.7 <http://dev.mysql.com/doc/refman/5.7/en/upgrading-from-previous-series.html>`_
+  * :ref:`changed_in_8.0`
+  * `Upgrading MySQL <http://dev.mysql.com/doc/refman/8.0/en/upgrading.html>`_
+  * `Upgrading from MySQL 5.7 to 8.0 <http://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html>`_
 
 .. warning:: 
 
- Upgrade from 5.6 to 5.7 on a crashed instance is not recommended. If the server instance has crashed, crash recovery should be run before proceeding with the upgrade. 
+ Upgrade from 5.7 to 8.0 on a crashed instance is not recommended. If the server instance has crashed, crash recovery should be run before proceeding with the upgrade. 
```

Comment:
Stronger wording than "not recommended"

Reply:
Added information about the native partitioning and row format